action/information.go

```diff
@@ -0,0 +1,52 @@
+package action
+
+import (
+	"encoding/json"
+	"fmt"
+	"go-bootcamp/data"
```

Comment:
Just separate the packages using best practices:

Go core

Project Local 

Third-Party